doc/connectors/aircall.md

```diff
@@ -1,21 +1,22 @@
-# Aircall connector
+# AircallOAuth connector
 
-Aircall Connector using Bearer to handle all the OAuth connection
+This is a connector for [Aircall](https://developer.aircall.io/api-references/#endpoints)
+    using oAuth for authentication
+
+A quickstart script is available in toucan-connectors/oauth2_connector/quickstart/
```

Comment:
:+1:

Reply:
Btw I changed the location of the quickstart script, it's in doc/connectors/